jobbergate-docs/src/developer_guide/s3_file_storage.rst

```diff
@@ -1,57 +0,0 @@
-=================
- S3 File Storage
-=================
-
-Besides the information kept in the database, application and job-scripts files are
-stored in an S3 Bucket.
-
-The bucket is named according to the ``S3_BUCKET_NAME`` environment variable used by
-the Jobbergate API.
-
-Within the the bucket, there should be two top-level folders, and the structure should
-appear similar to:
-
-.. code-block::
```

Comment:
I do not think this was migrated to markdown, but I want to double-check since it is pretty outdated.

Reply:
Yes, I just dropped this whole document. It is very updated. We can add a new version of it later if we wish, but I didn't want to take the time to rewrite this entirely.